uportal-war/src/main/data/quickstart_entities/fragment-layout/guest-lo.fragment-layout.xml

```diff
@@ -36,6 +36,7 @@
             <channel fname="personalization-gallery" unremovable="false" hidden="false" immutable="false" ID="n42"/>
         </folder>
         <folder ID="s100" hidden="false" immutable="true" name="Pre Header folder" type="pre-header" unremovable="true">
+            <channel fname="portal-menu-toggle" unremovable="false" hidden="false" immutable="false" ID="n9"/>
```

Comment:
Probably could simply add to `all-lo` (instead of 2 places);  fragment priority can control ordering.